bitcoin-magazine/bitcoin-explained/accounts-for-bitcoin-easypaysy!.md

```diff
@@ -0,0 +1,1038 @@
+
+Sjors: 00:13:32
+
+For example, the block height plus the place in the block.
+
+Aaron: 00:13:35
+
+That's exactly what it is.
+Not for example, that's exactly what it is.
+So the block height is, I don't know, 578331.
+I'm just making something up.
+
+Sjors: 00:13:47
+
+You're one of those old schoolers that still remembers blocks in the 500,000 range.
+
+Aaron: 00:13:52
+
+No, I think that's what I used in my article when I wrote this.
+So that's why it's still sort of stuck in my head.
+I didn't use this exact number, but somewhere in the 500k range, I think.
+And then it's like the 738th in the blockchain and then the actual account identifier is, if I'm recalling correctly, it starts with the actual blockchain.
+So if it's Bitcoin, it's like BTC at 500, whatever number I just used in the 500k range dot and then 700 whichever one I just used and then I think there's a slash and there's a checksum.
+So then you have this identifier, which is based on the place in the blockchain.
+There are ways where you can revert.
+That's not the right word.
+
+Sjors: 00:14:45
+
+I think what you were going to say is it's going to convert to some nice mnemonic style.
+
+Aaron: 00:14:49
+
+Yeah.
+
+Sjors: 00:14:50
+
+So a couple of words like rabbit, big, fish.
+
+Aaron: 00:14:54
+
+Exactly.
+Right now I was unable to remember the exact numbers I used, but if I would have used a mnemonic in the first place, Rabbit, fish, I already forgot.
+
+Sjors: 00:15:03
+
+Yeah, rabbit, big fish.
+
+Aaron: 00:15:04
+
+Whatever it was.
+That's much easier to remember.
+
+Sjors: 00:15:07
+
+Yeah that makes sense.
+
+Aaron: 00:15:08
+
+So then you can remember it and you can tell people and you know, you don't forget.
+It's much easier to remember.
+
+Sjors: 00:15:13
+
+And you can go on TV and say, you know, remember rabbit, big, fish.
+Then people can send money to rabbit, big, fish.
+And maybe that's slightly easier than giving them an address, though I'm a little skeptical whether that short name is such a big deal.
+But this is kind of where we get privacy and also the stealth address part.
+
+Aaron: 00:15:31
+
+Let's hear it.
+
+Sjors: 00:15:32
+
+So the first time you're using this as a recipient, so you're going to create an account, you're going to put a transaction on the blockchain.
+If then later on you receive coins, we're going to assume you use this third methodology, the nice privacy thing, it's going to arrive on different addresses.
+So that's very nice.
+But, if you're stupid, or if your wallet is stupid, it's going to combine coins that, you know, are related to that first transaction.
+The first transaction itself is not spendable because it's an OP_RETURN, but maybe it comes from another coin.
+
+Aaron: 00:16:04
+
+Sure.
+
+Sjors: 00:16:05
+
+And now you start combining coins and somebody can link your identity to everything you received.
+So you still need to be very wary of this cluster analysis risk.
+
+Aaron: 00:16:14
+
+Yeah, you need to...
+
```

Comment:
This kind of lines don't contribute to the transcript's content - you can just delete this and continue with the same speaker. This transcript has a lot back n forth between participants, so you'll notice the presence of numerous filler words or non-essential lines that need removal.